packages/cli/src/commands/build.ts

```diff
@@ -114,7 +114,9 @@ export async function build({
   runtime.on(Events.PreStepExecute, (t, n, _c, d) => console.log(`${'  '.repeat(d)}exec: ${t}.${n}`));
   runtime.on(Events.SkipDeploy, (n, err, d) => {
     partialDeploy = true;
-    console.log(`${'  '.repeat(d)}  -> skip ${n} (${err.toString()})`);
+    console.log(
+      `${'  '.repeat(d)}  -> skip ${n} (${err.toString() === '[object Object]' ? JSON.stringify(err) : err.toString()})`
```

Comment:
@dbeal-eth Can you try this? if it works would support more general types
```suggestion
      `${'  '.repeat(d)}  -> skip ${n} (${typeof err === 'object' ? JSON.stringify(err) : err.toString()})`
```

Reply:
im just worried that objects that have a proper `toString` implementation will end up still getting stringified even though they printo ut something resasonable/desired for `toString()`